de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/editor/StoppableDocumentListener.java

```diff
@@ -57,9 +54,10 @@ public void beforeDocumentChange(DocumentEvent event) {
                 return;
             }
 
-            path = virtualFileConverter.convertToPath(virtualFile);
+            path = VirtualFileConverter.convertToSPath(virtualFile);
+
+            if (path == null || !SessionUtils.isShared(path)) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "SessionUtils"; it is deprecated. ([squid:CallToDeprecatedMethod](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3ACallToDeprecatedMethod))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=294&issues=AWcxYNBo0EcSutaxLwSv&open=AWcxYNBo0EcSutaxLwSv)

[ISSUE_KEY]: AWcxYNBo0EcSutaxLwSv